features/view_specs/view_spec.feature

```diff
@@ -212,3 +212,24 @@ Feature: view spec
       """
     When I run `rspec spec/views`
     Then the examples should all pass
+
+  Scenario: path helpers should take precedence over asset helper
```

Comment:
Yes but consider the case where someone is testing that an output matches an asset path

```
it "displays my video" do
  expect(view).to include video_path('my_asset')
end
```